src/main/flight/pid_luxfloat.c

```diff
@@ -62,6 +62,10 @@ extern float lastITermf[3], ITermLimitf[3];
 extern pt1Filter_t deltaFilter[3];
 extern pt1Filter_t yawFilter;
 
+extern bool dtermNotchInitialised, dtermBiquadLpfInitialised;
```

Comment:
done, see 0e22db7ec16ca5a2b01b9d8276ced5e5966195b7 and 3b233ddc98afc5ba6bdfd14eb0b2ffc34cb7af56